broadcast/MintOnL2.s.sol/43113/run-1686792785.json

```diff
@@ -0,0 +1,198 @@
+{
```

Comment:
*arbitrary comment location*

Wondering out loud here whether there is actually any value to checking in all these broadcast files. What do you think, is there a reason we'd want them all?

Reply:
My reasoning is we should only check in the deployments used in the specialized relayer or of a deployed version that is merged into main. That way we can easily find past deployments if we want to validate past functionality or revert to a previous "working state".

Reply:
Ok, that's fine. I'm pretty indifferent on it. Maybe after we get things to a stable working version we can come back around and clean up old deployment records we're unlikely to need/want again. Thanks!